packages/odyssey-design-tokens/src/shadow.json

```diff
@@ -1,5 +1,6 @@
 {
   "shadow": {
-    "base": { "value": "0 2px 0 rgba(175, 175, 175, 0.12)" }
+    "base": { "value": "0 0.125rem 0 rgba(175, 175, 175, 0.12)" },
+    "hover": { "value": "0 0.5rem 0.75rem rgba(29, 29, 33, 0.12)" }
```

Comment:
as an FYI [see my comment and threaded details here](https://okta.slack.com/archives/G01JS0FE32P/p1641850336023700)

Reply:
Widths here should remain `px`, as we don't want the shadow to increase in size with the type.

Are we using this as a standard outside of Box? This is another place Design hasn't solidified a token.

Reply:
@edburyenegren-okta I know we talked about this in slack a bit but I disagree that we should be using `px` units. There are a few good reasons for this. @josesolano-okta was on board with `rem` units and brought up two pieces of feedback I can address here (and will link to in slack):

* Default sizes should remain intact: this is still true. Each rem unit in my PR here maps to the corresponding previous pixel value under a 16px browser default.

* Ability to adjust font size and border in isolation. This is also possible via the theming system. A user can increase the font size and change the hover shadow token here independently.

Reply:
What we don't achieve with `rem` is that Design wants the values to stay the same regardless of font-size.

Ex: If an Okta product has 12px, 14px, or 16px (Workflows, Admin, EndUser), the shadow should look the same without the team needing to adjust. 

Similarly, if a user bumps their font-size (in browser) to something like 24px for a11y reasons, we don't want the shadows to be 150% as large.